### ISSUE TITLE
Term Request: Sawtooth pattern of Bowman layer

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11742]
**Preferred term label:**
Sawtooth pattern of Bowman layer

**Synonyms**
* Saw-tooth Bowman layer configuration
* Wavy saw-toothed subepithelial pannus
* Sawtooth configuration on anterior segment OCT

**Definition (free text, please give PubMed ID)**
An undulating, serrated, or saw-tooth configuration of hyperreflective fibrocellular material replacing the Bowman layer and projecting into the overlying basal epithelium, pathognomonic on anterior segment optical coherence tomography (AS-OCT) or histology (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Subepithelial corneal opacities (HP:0008039)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Thiel-Behnke corneal dystrophy (TBCD) (ORPHA:98960, OMIM:602082)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
